core/src/hash_table/actor.rs

```diff
@@ -23,51 +23,63 @@ impl HashTable for ActorRef<Protocol> {
         unwrap_to!(response => Protocol::TeardownResult).clone()
     }
 
-    fn put_pair(&mut self, pair: &Pair) -> Result<(), HolochainError> {
-        let response = self.block_on_ask(Protocol::PutPair(pair.clone()));
-        unwrap_to!(response => Protocol::PutPairResult).clone()
+    fn put_entry(&mut self, entry: &Entry) -> Result<(), HolochainError> {
```

Comment:
This makes it look like what goes in the hash table is entries, and is part of the confusion that leads there to be non-chain "entry" types and has led to the follow-up ticket on ["dht system entry types"](https://github.com/holochain/holochain-rust/issues/339)  I'm ok with approving this PR because we need to move all of the parts forward, but I'm just calling this out here for us to remember that the word "Entry" I think should be reserved for what can be added to the chain.  In the current code there are "DHT only" entry types for holding groups of links for example.   This will cause confusion and make us think we can commit such items to the chain, which we shouldn't be able to.

Reply:
Ok. I'll add a todo here as a reminder.